src/cryptoadvance/specter/helpers.py

```diff
@@ -322,3 +322,11 @@ def is_ip_private(ip):
         or priv_16.match(ip)
     )
     return res is not None
+
+
+def get_address_from_dict(data_dict):
+    return (
+        data_dict["addresses"][0]
+        if "addresses" in data_dict and data_dict["addresses"][0]
+        else data_dict["address"]
```

Comment:
Maybe put a comment here to drop `["addresses"][0]` when you drop support for Bitcoin Core < v22 in the distant future.

Reply:
Makes sense, added https://github.com/cryptoadvance/specter-desktop/pull/1180/commits/fbab6eba2d2be0f09430765a3574a8f8f0dacab4#diff-2c38e061a909d8eb0a1d1e836cb1d04d93578e9c53d98841eb5a13edb0b45fa6R328